Utilities/src/main/java/edu/stanford/hivdb/mutations/Apobec.java

```diff
@@ -113,13 +107,24 @@ public String generateComment() {
 		return comment.toString();
 	}
 
+	static void staticInit() {
+		Cachable.setup(Apobec.class, () -> {
+			try {
+				populateApobecMaps();
+			} catch (SQLException e) {
+				throw new ExceptionInInitializerError(e);
+			}
+		});
+	}
+	
 	// Populates two sets. One containing all mutations indicative
 	// of APOBEC-mediated G-to-A hypermutation.
 	// The second containing those DRMs that could be selected by
 	// therapy or could could be caused by APOBEC.
-	private static void populateApobecMaps() throws SQLException {
-		final JdbcDatabase db = JdbcDatabase.getDefault();
-
+	//
+	// Note: This function is package-private for testing, but it should
+	// otherwise be invoked privately.
+	public static void populateApobecMaps() throws SQLException {
```

Comment:
Use `protected` if you want to access this function from test class.

Reply:
Good call.